src/plots/cartesian/layout_attributes.js

```diff
@@ -180,7 +180,7 @@ module.exports = {
         description: [
             'Sets the text displayed at the ticks position via `tickvals`.',
             'Only has an effect if `tickmode` is set to *array*.',
```

Comment:
It depends what we mean by _set_.

Does _set_ mean:

1. that `tickmode` **has to be set** by the user to `'array'`, or
1. that `tickmode` is **either** set by the user to `'array'` or inferred to be `'array'` by [`tickvals`](https://github.com/plotly/plotly.js/blob/071474781273374eef4beca0298aed60929a2fe6/src/plots/cartesian/tick_value_defaults.js#L23).

I think item 1. is probably what most people would understand by _set_. But how should we describe situation similar to item 2. ?